test/config.json

```diff
@@ -210,6 +210,17 @@
 				"site_test"
 			]
 		},
+		"parser_generator": {
+			"File": "",
+			"Args": [],
+			"Command": [
+				"tools/test_parser_generator.sh"
```

Comment:
Right now the name suggests that we would test the "parser generator". That may be misleading here if our only intent is to verify that the generated code is in sync?

For protos we already have such a check which we called "check_make_proto.sh": https://github.com/youtube/vitess/blob/master/travis/check_make_proto.sh

(That file should be moved out of the "travis" directory to the "tools" directory which you use.)

For consistency we could call it "check_make_parser.sh"?